### PR TITLE
[NUI] Add AttributeUsageAttribute for ScriptableProperty(CA1018)

### PR DESCRIPTION
**Pull request by dasg34**
### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/CustomViewRegistry.cs [hunk 1]
```diff
@@ -72,6 +72,7 @@ namespace Tizen.NUI
     ///
     ///
     /// <since_tizen> 3 </since_tizen>
+    [AttributeUsage(AttributeTargets.Property)]
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/CustomViewRegistry.cs:75]: I am not sure that by this change, another warning has been shown as like as below? 

`src/Tizen.NUI/src/public/CustomViewRegistry.cs#L76
Rename Tizen.NUI.ScriptableProperty to end in 'Attribute'.`

it asks to rename the class name as it should have the "Attribute" suffix.
is this another type of warning? or is this related to this?

Comment (dasg34) [reply src/Tizen.NUI/src/public/CustomViewRegistry.cs:75]: CA1710(`Rename Tizen.NUI.ScriptableProperty to end in 'Attribute'.`)  is not related with this patch.
This patch is only for fixing CA1018(`Specify AttributeUsage on ScriptableProperty.`).

**Comment by dasg34** [comment]
ping~

**Review by dongsug-song — APPROVED** [review]
looks good to me
